content/applications/finance/fiscal_localizations/belgium.rst

```diff
@@ -377,70 +366,49 @@ Once the connection is established, Odoo can be synchronized with CodaBox.
-    the link on Odoo's side only.
-
-    If you have multiple connections, you will still be able to use the
-    Access Token of one of them to easily reconnect Odoo and CodaBox for
-    another client.
-
-    However, if you have only one (remaining) connection, you will not be able
-    to use its Access Token to create new connections. In that case, you will
-    have to revoke the connection from the CodaBox platform too,
-    and then recreate a new one following the steps in the
-    :ref:`first connection <belgium/codabox-configuration-connection>` section.
+    The connection between Odoo and CodaBox can be revoked by going to
+    :menuselection:`Accounting --> Configuration --> Accounting: CodaBox --> Manage Connection`
+    and clicking on the :guilabel:`Revoke` button.
 
 Potential issues
 ~~~~~~~~~~~~~~~~
 
 CodaBox is not configured. Please check your configuration.
   Either the Company VAT of the Accounting Firm VAT is not set.
 
-It seems that the fiduciary VAT number you provided is not valid. Please check your configuration.
-  Either the Company VAT of the Accounting Firm VAT is in the wrong format.
+No connection exists with these fiduciary and company VAT numbers. Please check your configuration.
+  This can happen when checking the connection status, and the pair (Fiduciary VAT, Company VAT)
+  is not registered yet. This may happen if you have changed the company VAT after the connection
+  was established. For security reasons, you have to recreate a connection for this company VAT.
+
+It seems that your CodaBox connection is not valid anymore. Please connect again.
+  This can happen if you have revoked the access of Odoo to your CodaBox account, or that you have
+  not completed the configuration process. In this case, you must revoke your connection and create
+  a new one.
+
+It seems that you have already created an Odoo-CodaBox connection with this fiduciary. You must reuse the password you received from Odoo during your first connection.
+  You must use the password you received from Odoo during your first connection to create a new
+  connection for this fiduciary. If you have lost your password, you will have to first revoke the
+  Odoo connection on CodaBox's side (i.e. on your myCodaBox portal). Then, you will be able to
+  revoke the connection on Odoo's side and create a new one.
+
+The provided password is not valid for this fiduciary.
+    The password you provided is not the one you received from Odoo during your first connection.
+    You must use the password you received from Odoo during your first connection to create a new
+    connection for this fiduciary. If you have lost your password, you will have to first revoke the
+    Odoo connection on CodaBox's side (i.e. on your myCodaBox portal). Then, you will be able to
+    revoke the connection on Odoo's side and create a new one.
+
+It seems that the company or fiduciary VAT number you provided is not valid. Please check your configuration.
```

Comment:
The company or accounting firm VAT number you provided is not valid. Please check the configuration.